extensions/ql-vscode/src/interface.ts

```diff
@@ -561,24 +561,26 @@ export class InterfaceManager extends DisposableObject {
     sourceInfo: cli.SourceInfo | undefined,
     sourceLocationPrefix: string,
     sortState: InterpretedResultsSortState | undefined
-  ): Promise<Interpretation> {
+  ): Promise<Interpretation | undefined> {
     const sarif = await interpretResults(
       this.cliServer,
       metadata,
       resultsPaths,
       sourceInfo
     );
+
+    if (!resultsPaths) {
```

Comment:
Should we do this before the call to `interpretResults` above? Or does that populate `resultsPaths`?

Reply:
Yes, I think this should be moved up.